dlt/common/runtime/collector.py

```diff
@@ -170,6 +170,7 @@ def update(
                 total=total,
             )
             self.messages[counter_key] = None
+            self.last_log_time = None
```

Comment:
> Load job counters were incorrectly increased (absolute value was used).

How does adding this line solve the issue of incorrectly increasing the counter?